js/packages/teams-ai/src/authentication/TeamsSsoMessagingExtensionAuthentication.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { TurnContext } from "botbuilder-core";
+import { TeamsChannelAccount, TokenResponse } from "botframework-schema";
+import { MessagingExtensionAuthenticationBase } from "./MessagingExtensionAuthenticationBase";
+import { TeamsSsoPromptSettings } from "./TeamsBotSsoPrompt";
+import { ConfidentialClientApplication } from "@azure/msal-node";
+import { TeamsInfo } from "botbuilder";
+import { MessagingExtensionsInvokeNames } from '../MessageExtensions';
+
+export class TeamsSsoMessagingExtensionAuthentication extends MessagingExtensionAuthenticationBase {
```

Comment:
Please rename to `TeamsSsoMessageExtensionAuthentication` (same with any non-external package instances that say MessagingExtension`